libs/c/include/stddef.h

```diff
@@ -5,4 +5,6 @@
 
 typedef unsigned int size_t;
 
+#define offsetof(type, member) (size_t) &((type *) NULL)->member
```

Comment:
I don't think it's really necessary though so feel free to ignore
```suggestion
#define offsetof(type, member) ((size_t) &((type *) NULL)->member)
```